grails-web-url-mappings/src/test/groovy/grails/web/mapping/GroupedUrlMappingSpec.groovy

```diff
@@ -27,30 +27,30 @@ import javax.servlet.http.HttpServletResponse
  */
 class GroupedUrlMappingSpec extends AbstractUrlMappingsSpec {
 
-    @Issue('#10308')
+    @Issue('#10308 #10844')
     void "Test mapping with group and nested collection"() {
         given:
         def linkGenerator = getLinkGenerator {
-            "/foos"(resources: 'foo') {
+            '/api1/employees'(resources: 'employee') {
                 collection {
-                    '/baz'(controller: 'foo', action: 'baz')
+                    '/search'(controller: 'employee1', action: 'search1', method: 'GET')
                 }
             }
 
-            group "/g", {
-                "/bars"(resources: 'bar') {
+            group '/api2', {
+                '/employees'(resources: 'employee') {
                     collection {
-                        '/baz'(controller: 'bar', action: 'baz')
+                        '/search'(controller: 'employee2', action: 'search2', method: 'GET')
                     }
                 }
             }
         }
 
         expect:
-        linkGenerator.link(controller:'bar', action:'baz', params:[barId:1]) == 'http://localhost/g/bars/1/baz'
+        linkGenerator.link(controller: 'employee1', action: 'search1', method: 'GET', params: [userId: 1]) == 'http://localhost/api1/employees/search?userId=1'
+        linkGenerator.link(controller: 'employee2', action: 'search2', method: 'GET', params: [userId: 1]) == 'http://localhost/api2/employees/search?userId=1'
     }
```

Comment:
Perhaps add a test with `employeeId` to verify it puts it in the URL?

Reply:
I think that's not possible. According to http://docs.grails.org/3.3.2/guide/theWebLayer.html#restfulMappings in `Nested Resources` section, that mapping is not created automatically.

Reply:
I give it a second thought and also read the documentation twice and I've push a commit.